Makefile

```diff
@@ -24,12 +24,12 @@ clean:
 
 build:
 	docker run -w /app -v ${ROOT}:/app ${GOLANG_DOCKER_IMAGE} go mod download
-	docker run -w /app -v ${ROOT}:/app ${GOLANG_DOCKER_IMAGE} go build cmd/cli/airtbl.go
 
 #   Deletes container if exists
 #   Usage:
 #       make run arguments="-help"
 run:
+	docker run -w /app -v ${ROOT}:/app ${GOLANG_DOCKER_IMAGE} go build cmd/cli/airtbl.go
```

Comment:
PR says that `make run` **used** to download go deps and build `airtbl.go` internally. 
This line is now building `airtbl.go` under `make run` right?

Reply:
nvm I see that build is the one doing the downloading, and run is just building and running